src/plays/keeper/Keeper.jsx

```diff
@@ -0,0 +1,78 @@
+
+import { useLocation } from 'react-router-dom';
```

Comment:
useLocation, are we using anywere? If not remove.

Reply:
Removed useLocation from Keeper.jsx